lms/djangoapps/ccx/views.py

```diff
@@ -22,6 +23,7 @@
 from django.core.validators import validate_email
 from django.http import Http404
 from django.shortcuts import redirect
+from django.core.paginator import Paginator, EmptyPage, PageNotAnInteger
```

Comment:
nit: this section seems to be sorted, so please move this line in the right position